jQuery/index.html

```diff
@@ -9,12 +10,20 @@
     <div id="hh-menu">
       <h1>Honey Headquarters</h1>
 
-      <label for="character-select">Character</label>
-      <select id="character-select"></select>
-      <label for="hometown-select">Hometown</label>
-      <select id="hometown-select"></select>
-      <label for="size-select">Size</label>
-      <select id="size-select"></select>
+      <label class="menu-label">
```

Comment:
This is much better. You may choose to add classes or give more descriptors if you think they'll help :)